script/build.js

```diff
@@ -35,6 +35,8 @@ const trashPaths = [
 ]
 
 const webpackArgs = [
+  "-r",
+  "esm",
```

Comment:
The `esm` referenced here is grabbing the one used by `ava`. It may not be doable to use `esm` to build `esm` without relying on a prepublished `esm`.

Reply:
I was already wondering where the reference came from :smiley:

Reply:
regarding chakra: gotcha. was about to file an issue :)

Reply:
Since there's a chicken and egg gotcha I think I'll punt on this for now.